services/minespace-web/src/components/Forms/noticeOfDeparture/EditNoticeOfDepartureForm.js

```diff
@@ -25,13 +25,14 @@ const propTypes = {
   onSubmit: PropTypes.func.isRequired,
   closeModal: PropTypes.func.isRequired,
   handleSubmit: PropTypes.func.isRequired,
+  pristine: PropTypes.bool.isRequired,
   mineGuid: PropTypes.string.isRequired,
   noticeOfDeparture: CustomPropTypes.noticeOfDeparture.isRequired,
 };
 
 // eslint-disable-next-line import/no-mutable-exports
 let AddNoticeOfDepartureForm = (props) => {
-  const { onSubmit, closeModal, handleSubmit, mineGuid, noticeOfDeparture } = props;
+  const { onSubmit, closeModal, handleSubmit, mineGuid, noticeOfDeparture, pristine } = props;
```

Comment:
How is `pristine` manupulated?

Reply:
It's a bool provided directly from Redux-Form that reports weather any of the fields have been touched.  I haven't played with manipulating it yet though.

Reply:
Ah, its an inbuilt functionality. thanks